app/lib/widgets/cards/productCard/shared/price.dart

```diff
@@ -0,0 +1,127 @@
+import 'package:flex_ui/utils/extensions.dart';
+import 'package:flutter/material.dart';
+import 'package:widgetbook/widgetbook.dart';
+import 'package:widgetbook_annotation/widgetbook_annotation.dart' as widgetbook;
+import 'package:intl/intl.dart';
+
+enum Variant {
+  standard,
+  savings, // Defaults to text decoration linethrough and color 'disabled'
+}
+
+class Price extends StatelessWidget {
```

Comment:
Suggestion: FlexPrice instead of Price. I think it's important to distinguish our widgets from Flutter's (and avoids issues : FlexAppbar / Appbar from Flutter)